tutoroars/plugin.py

```diff
@@ -34,13 +34,7 @@
 
         # Make sure LMS / CMS have evnet-routing-backends installed
         # TODO: Do a new release and pin this! Also add config!
-        ("OPENEDX_EXTRA_PIP_REQUIREMENTS", "event-routing-backends"),
-
-        # Demo data (optional)
-        # set to "xapi" to load xapi demo data into clickhouse
-        ("OARS_CLICKHOUSE_LOAD_DEMO_DATA", "no"),
-        ("OARS_CLICKHOUSE_LOAD_DEMO_XAPI_BATCHES", "100"),
-        ("OARS_CLICKHOUSE_LOAD_DEMO_XAPI_BATCH_SIZE", "100"),
+        ("OPENEDX_EXTRA_PIP_REQUIREMENTS", ["edx-event-routing-backends"]),
```

Comment:
I think that if this went into overrides it would prevent other plugins from adding requirements? I believe doing it this way is additive so won't overwrite the default extra requirements, for instance.

Reply:
Oo neat, I wondered how the additive stuff was supposed to work. Thanks!